onnxruntime/python/tools/kernel_explorer/kernels/gemm_test.py

```diff
@@ -71,6 +72,10 @@ def _test_gemm(func, dtype: str, transa: bool, transb: bool, m: int, n: int, k:
 all_transabs = list(product([True, False], repeat=2))
 
 
+@pytest.mark.skipif(
+    not int(os.environ.get("KERNEL_EXPLORER_TEST_VENDOR_LIBRARIES", "0")),
```

Comment:
> KERNEL_EXPLORER_TEST_VENDOR_LIBRARIES

Naming: This feels like a very generic flag that indicates that _all_ vendor lib tests will be skipped if set to 0. However, you're only skipping rocblas tests (I think?). What about CK, and possibly hipBLASLT when we add those? Would we want to skip those as well? 

For now, I would suggest naming it `KERNEL_EXPLORER_SKIP_ROCBLAS_TESTS` or something similar to indicate the narrow scope of this variable.